packages/grafana-ui/src/components/DataSourceSettings/BasicAuthSettings.tsx

```diff
@@ -31,9 +35,11 @@ export const BasicAuthSettings: React.FC<HttpSettingsProps> = ({ dataSourceConfi
     });
   };
 
+  const styles = useStyles(getStyles);
+
   return (
     <>
-      <div className="gf-form">
+      <div className={styles.form}>
```

Comment:
I could be wrong but I believe the `<InlineField>` component can be used in place of these custom styles and the legacy `<FormField>`. 🤔

Reply:
Yea this should be InlineField, we should not duplicate these styles. The gf-form classes we need to preserve as they are heavily used by old plugins.